.github/workflows/run_ec2_pcluster.yaml

```diff
@@ -135,38 +136,35 @@ jobs:
           make -j8
 
           # Runs mandatory ctests on the head node
-          ctest -R get_
-          ctest -R fv3jedi_setup_ufs_rundir_warmstart
-          ctest -R fv3jedi_setup_ufs_rundir_coldstart
-          ctest -R fv3jedi_test_tier1_model_ufs_warmstart
-          #ctest -R fv3jedi_test_tier1_model_ufs_coldstart
-          ctest -R fv3jedi_test_tier1_forecast_ufs_warmstart
-          #ctest -R fv3jedi_test_tier1_forecast_ufs_coldstart
+          rm -rf fv3-jedi/test/Data/ModelRunDirs/UFS_warmstart/*
+          rm -rf fv3-jedi/test/Data/ModelRunDirs/UFS_warmstart_2/*
+          ctest -R ufs_
 
           # UFS_APP=ATMAERO
           cd ${JEDI_ENV}
           mkdir -p build-atmaero
           cd build-atmaero
+          rm -rf *
```

Comment:
Why is this needed?

Reply:
I had to put it there to make sure the build was started fresh. I'll remove it now.